screens/OrderFormII.js

```diff
@@ -72,8 +71,15 @@ class OrderFormII extends React.Component {
   componentDidMount() {
     this.props.getLables()
     this.props.getProducts()
-    this.props.navigation.state.params.orderId !== undefined &&
-      this.props.getOrder(this.props.navigation.state.params.orderId)
+    console.log(this.props.route)
```

Comment:
Remove

Reply:
Sorry, it's. a test miss. I will remove console and recover getOrder().